modules/private-clusters-setting-api-private.adoc

```diff
@@ -15,22 +30,65 @@ After you deploy a cluster to Amazon Web Services (AWS) or Microsoft Azure, you
 
 .Procedure
 
-. In the web portal or console for AWS or Azure, take the following actions:
+. In the web portal or console for your cloud provider, take the following actions:
 
-.. Locate and delete appropriate load balancer component.
+.. Locate and delete the appropriate load balancer component:
+ifndef::cpmso-using-azure[]
 *** For AWS, delete the external load balancer. The API DNS entry in the private zone already points to the internal load balancer, which uses an identical configuration, so you do not need to modify the internal load balancer.
+endif::cpmso-using-azure[]
+ifndef::cpmso-using-aws[]
 *** For Azure, delete the `api-internal` rule for the load balancer.
+endif::cpmso-using-aws[]
 
 .. Delete the `api.$clustername.$yourdomain` DNS entry in the public zone.
 
+ifndef::post-install[]
+. Remove the external load balancers by deleting the following lines in the control plane machine set custom resource:
++
+[source,yaml]
+----
+...
+spec:
+  providerSpec:
+    value:
+    ...
+      loadBalancers:
+      - name: lk4pj-ext <1>
+        type: network <1>
+      - name: lk4pj-int
+        type: network
+----
+<1> Delete this line.
+endif::post-install[]
+
+ifdef::post-install[]
 . Remove the external load balancers:
 +
 [IMPORTANT]
 ====
 You can run the following steps only for an installer-provisioned infrastructure (IPI) cluster. For a user-provisioned infrastructure (UPI) cluster, you must manually remove or disable the external load balancers.
 ====
 +
-.. From your terminal, list the cluster machines:
+** If your cluster uses a control plane machine set, delete the following lines in the control plane machine set custom resource:
++
+[source,yaml]
+----
+...
+spec:
```

Comment:
There's a few levels missing here, does that matter?

Reply:
As long as they are not levels needed for the procedure, should be ok. This is was stolen from the existing docs, but if it needs an update to be clear lmk and I'm happy to do it :)